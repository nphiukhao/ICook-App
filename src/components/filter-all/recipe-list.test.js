import React from 'react'
import Recipelist from './recipe-list'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe(`Recipe list component`, () => {
    it('Recipe list snapshot test', () => {
        const tree = shallow(<Recipelist/>)
        expect(toJson(tree)).toMatchSnapshot()
    })
})