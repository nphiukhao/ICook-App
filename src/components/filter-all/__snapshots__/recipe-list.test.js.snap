// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Recipe list component Recipe list snapshot test 1`] = `
<div
  className="recipe-links-container"
>
  <Link
    style={
      Object {
        "textDecoration": "none",
      }
    }
    to="/recipe/undefined"
  >
    <h3 />
    <h4>
      Total Cooking Time: 
       mins.
    </h4>
  </Link>
</div>
`;
